assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -73,8 +74,14 @@ export default function useMediaPicker({
       // The Uploader.success callback is invoked when a user uploads a file.
       // Race condition concern: the video content is not guaranteed to be
       // available in this callback. For the video poster insertion, please check: assets/src/edit-story/components/library/panes/media/local/mediaPane.js
+      const trackTiming = getTimeTracker(
+        'updateMedia',
+        'MediaPicker',
+        'Media Library'
+      );
       wp.Uploader.prototype.success = ({ attributes }) => {
         updateMedia(attributes.id, { media_source: 'editor' });
+        trackTiming();
```

Comment:
Two things:

1. This is not really worth tracking IMO, all it would track is how long it takes to do this `updateMedia` call. Not something I'd be interested in
2. This does not work as intended. This starts the timer, then defines a function on the prototype, and then stops the timer once the function is called and finished.  
  Instead, the timer would need to be started within the function itself

Reply:
Sure, I will remove the check here.